src/chainstate/stacks/address.rs

```diff
@@ -92,6 +94,34 @@ pub enum PoxAddress {
     Addr32(bool, PoxAddressType32, [u8; 32]),
 }
 
+pub fn display_serialize<S: Serializer, T: std::fmt::Display>(
+    input: &T,
+    ser: S,
+) -> Result<S::Ok, S::Error> {
+    ser.serialize_str(&input.to_string())
+}
+
+pub fn address_deser<'de, D: Deserializer<'de>, T: Address>(deser: D) -> Result<T, D::Error> {
```

Comment:
Nit: rename to `address_deserialize` to be consistent with other deserialization function names in the codebase.

Reply:
This method was removed in 92ec39e, because it uses `PrincipalData` deserialization instead. That method is called `principal_deserialize`